### ISSUE TITLE
Error if constructor doesn't call super()

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#466]
This code should report an error:
```js
class MessageFetcher extends EventEmitter {
  constructor() {
  }
}
```

**Comment by strager** [comment]
Are you still working on this task, @H4ckOm?

**Comment by H4ckOm** [comment]
Hi @strager I'm not working on this task now, I got busy somewhere else.
